### DISCUSSION TITLE
Highlight the visibility of  memos

### DISCUSSION THREAD

**Discussion by sogg** [usememos/memos#4129] [Ideas]
Thank you to the developers. I've been using Memos for a long time and it's been a great experience. I have an idea (or a requirement), and I'm not sure if it can get support:
The visibility of Memos is divided into "public", "private" and "workspace", but it's not eye-catching enough. The icon is only displayed when the mouse hovers over the upper right corner of each memo. Could you add a class attribute related to visibility to the div of Memos and set different display styles? For example, 'memo-visable-private','memo-visable-public','memo-visable-workspace'.
If anyone knows how to enhance the highlighting of visibility through custom CSS, please give me some guidance.

Thanks Again.

**Comment by RoccoSmit** [comment]
You can go to `settings` -> `system` -> `additional styles` and add the following

```
.group.relative.flex.flex-col.justify-start.items-start.w-full.px-4.py-3.mb-2.gap-2:has(div div span .lucide-earth) {
    background-color: rgba(0, 255, 62, 0.1); // public
}

.group.relative.flex.flex-col.justify-start.items-start.w-full.px-4.py-3.mb-2.gap-2:has(div div span .lucide-users) {
    background-color: rgba(0, 47, 255, 0.1); // workspace
}

.group.relative.flex.flex-col.justify-start.items-start.w-full.px-4.py-3.mb-2.gap-2 {
    background-color: rgba(255, 0, 0, 0.1); // private
}
```

You can change the colours to what you prefer
e.g. of the above

![image](https://github.com/user-attachments/assets/fd60381f-83a8-415a-a544-8c43310e2748)

**Reply by sogg** [reply]
Perfect! that  works. Thank you very much.
